FreeRTOS-Plus/Source/Application-Protocols/network_transport/sockets_wrapper/cellular/sockets_wrapper.c

```diff
@@ -89,9 +89,6 @@ extern uint8_t CellularSocketPdnContextId;
 #define CELLULAR_SOCKET_OPEN_TIMEOUT_TICKS     ( portMAX_DELAY )
 #define CELLULAR_SOCKET_CLOSE_TIMEOUT_TICKS    ( pdMS_TO_TICKS( 10000U ) )
 
-/* Cellular socket AT command timeout. */
-#define CELLULAR_SOCKET_RECV_TIMEOUT_MS        ( 1000UL )
-
 /* Time conversion constants. */
 #define _MILLISECONDS_PER_SECOND               ( 1000 )                                          /**< @brief Milliseconds per second. */
 #define _MILLISECONDS_PER_TICK                 ( _MILLISECONDS_PER_SECOND / configTICK_RATE_HZ ) /**< Milliseconds per FreeRTOS tick. */
```

Comment:
This macro would likely work better if the ms's were multiplied before dividing by the configTICK_RATE_HZ to avoid unnecessary loss of precision.